app/views/devise/sessions/new.html.slim

```diff
@@ -0,0 +1,21 @@
+.col-xs-10.col-xs-offset-1.col-sm-8.col-sm-offset-2.col-md-4.col-md-offset-4
+  .login-panel.panel.panel-default
+    .panel-heading
+      h2 Log in
+    .panel-body
+      = form_for(resource, as:  resource_name,
```

Comment:
use ```simple_form``` https://github.com/plataformatec/simple_form .
Your mates have already used the gem in their tasks.